### ISSUE TITLE
Tailscale suggests a mullvad exit node that is not available

### ISSUE THREAD

**Issue by ajgrf** [tailscale/tailscale#12311]
### What is the issue?

Running `tailscale exit-node suggest` returns a mullvad exit node that is not actually available to use as an exit node (us-lax-wg-201.mullvad.ts.net) for me. I think maybe I used this exit node in the past before it was removed or something.

I'm also using tailscale lock, but `tailscale lock status` does not show any nodes that still need to be signed.

### OS

Linux

### OS version

NixOS

### Tailscale version

1.66.4

### Bug report

BUG-3d4826d6e150193abbbf7b4ad3f69de0300d21a18b9b1f7901bc3fa08bb5d9a8-20240531205314Z-25116e720a3fc746

**Comment by sailorfrag** [comment]
The command printed out by the suggestion includes the wrong identifier for the exit node (#12618), but from the details provided I can't tell if this is the same problem or not. The bugreport logs do not include what you were doing with the CLI in this case. Can you provide a transcript of running the suggest command followed by trying to set the exit node?

**Comment by ajgrf** [comment]
Here you go, is this all you were after?

```
alex@staunch-coral:~$ tailscale exit-node suggest
Suggested exit node: us-lax-wg-405.mullvad.ts.net.
To accept this suggestion, use `tailscale set --exit-node=nCJRGm4CNTRL`.
alex@staunch-coral:~$ tailscale set --exit-node=nCJRGm4CNTRL invalid value "nCJRGm4CNTRL" for --exit-node; must be IP or unique node name
[ble: exit 1]
1|alex@staunch-coral:~$ 
```

**Comment by sailorfrag** [comment]
Yes, that's exactly what I needed, thanks!

What you're seeing is the same problem as #12618 and will be fixed in the next release. In the meantime,  please use `tailscale set --exit-node=us-lax-wg-405.mullvad.ts.net` (from the line above, remove the trailing dot) and it should work.

**Comment by sailorfrag** [comment]
Duplicate of #12618 

Please reopen this if the workaround does not work for you.

**Comment by sailorfrag** [comment]
To be more specific, it should be fixed in any unstable build at least 1.69.83 (it looks like the last unstable build is 1.69.66 but a new one will likely be built in the next few days), or stable 1.70.0.
